src/legacy/core_plugins/vis_default_editor/public/vis_options_props.tsx

```diff
@@ -27,7 +26,6 @@ export interface VisOptionsProps<VisParamType = unknown> {
   isTabSelected: boolean;
   stateParams: VisParamType;
   vis: Vis;
-  uiState: PersistedState;
```

Comment:
When you removed passing down the `uiState` into vis options tab, we lost next functionality:
- create a gauge vis;
- change a color for a metric in the legend (in options tab you'll see the `Reset colors` button) - the `uiState` should be preserved in the url;
- reload the page - you'll lost the actual ui state (color is lost, button is not shown)